pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/CountMVAggregationFunction.java

```diff
@@ -82,6 +104,20 @@ public void aggregateGroupBySV(int length, int[] groupKeyArray, GroupByResultHol
   public void aggregateGroupByMV(int length, int[][] groupKeysArray, GroupByResultHolder groupByResultHolder,
       Map<ExpressionContext, BlockValSet> blockValSetMap) {
     BlockValSet blockValSet = blockValSetMap.get(_expression);
+
+    if (blockValSet.isSingleValue()) {
```

Comment:
This code block should be removed as startree index won't be in effect when group by multi-value column, and should be removed?